tests/all.js

```diff
@@ -144,6 +144,22 @@ test('deep nested relative import/export', function(){
   equal(require('foo/a/b/c'), 'baz');
 });
 
+test('incorrect lookup paths should fail', function(){
+  expect(1);
+
+  define('foo/isolated-container', [], function() {
+    return 'container';
+  });
+
+  var willThrow = define('foo', ['./isolated-container'], function(container) {
+    return {
+      container: container
+    };
+  });
+
+  throws(willThrow);
```

Comment:
we should match the thrown expression correctly, to be sure it is the error we expect